upload-sourcemaps.js

```diff
@@ -81,6 +81,35 @@ module.exports = async (options) => {
 
     output = uploadResult.stdout.toString();
     log(output);
+
+    setCommits = setCommits || process.env.SENTRY_EXPO_SET_COMMITS;
+    if (setCommits) {
+      let commitsResult = await spawnAsync(
+        sentryCliBinaryPath,
+        ['releases', 'set-commits', '--auto', version],
+        {
+          env: childProcessEnv,
+        }
+      );
+
+      output = commitsResult.stdout.toString();
+      log(output);
+    }
+
```

Comment:
```suggestion
    let finalizeReleaseResult = await spawnAsync(
      sentryCliBinaryPath,
      ['releases', 'finalize', version],
      {
        env: childProcessEnv,
      }
    );
```

adding in finalizing the release to adhere to sentry's guide https://blog.sentry.io/2019/12/17/using-github-actions-to-create-sentry-releases